apps/condo/domains/ticket/access/CreateResidentTicketService.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Generated by `createservice ticket.CreateResidentTicketService --type mutations`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+async function canCreateResidentTicket ({ authentication: { item: user } }) {
+    // TODO(nomerdvadcatpyat): There will be a check for user type == 'resident' (after my pull request is accepted to master)
+    if (!user) return throwAuthenticationError()
+    return true
```

Comment:
add organization based checks

Reply:
No it’s ok :) you don’t need any checks for create tickets by residents.